units/INC9002/INC9002_unit.bp

```diff
@@ -0,0 +1,155 @@
+UnitBlueprint {
+    Audio = {
+        DeathExplosion = Sound {
+            Bank = 'XSL_Destroy',
+            Cue = 'XSB_Destroy_Lrg_PreDestroy',
+            LodCutoff = 'UnitMove_LodCutoff',
+        },
+        Destroyed = Sound {
+            Bank = 'XSL_Destroy',
+            Cue = 'XSB_Destroy_Huge',
+            LodCutoff = 'UnitMove_LodCutoff',
+        },
+        UISelection = Sound {
+            Bank = 'SeraphimSelect',
+            Cue = 'Seraphim_Select_Structure',
+            LodCutoff = 'UnitMove_LodCutoff',
+        },
+    },
+    Categories = {
+        'NOMAD',
+        'STRUCTURE',
+        'CIVILIAN',
+        'LAND',
+        'VISIBLETORECON',
+        'RECLAIMABLE',
+    },
+    CollisionOffsetX = 0,
+    CollisionOffsetY = 0,
+    CollisionOffsetZ = 0,
+    Defense = {
+        ArmorType = 'Normal',
+        Health = 5000,
+        MaxHealth = 5000,
+        RegenRate = 0,
+    },
+    Description = '<LOC inc9002_desc>Special Crystal',
```

Comment:
We could use some better name, objective to get some "special" crystals doesn't sound that good